runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerPlugin.java

```diff
@@ -54,4 +61,25 @@ public void shutDown()
 	{
 		overlayManager.remove(overlay);
 	}
+
+	@Subscribe
+	public void onVarClientIntChanged(VarClientIntChanged varClientIntChanged)
+	{
+		if (varClientIntChanged.getIndex() == VarClientInt.INVENTORY_TAB.getIndex())
+		{
+			if (isOnInvTab())
+			{
+				overlayManager.remove(overlay);
+			}
+			else
+			{
+				overlayManager.add(overlay);
+			}
+		}
+	}
+
+	private boolean isOnInvTab()
+	{
+		return client.getVar(VarClientInt.INVENTORY_TAB) == 3;
```

Comment:
I think we have a constant for this?

Reply:
I don't think there is. All other INVENTORY_TAB uses don't use any and I can't find a constant for this.